src/olympia/reviewers/tests/test_views.py

```diff
@@ -5326,29 +5336,24 @@ def test_important_changes_log(self):
 
         # Make sure the logs are displayed in the page.
         important_changes = doc('#important-changes-history li')
-        assert len(important_changes) == 6
-        # Note: addon_factory() tricks avoid some of the ActivityLogs we'd
-        # normally be getting for an approved add-on, because it forces the
-        # status - that's fine, in this case we only want to make sure the
-        # change status ones are included in the log.
-        assert ' status changed to Incomplete' in important_changes[0].text_content()
-        assert 'class' not in important_changes[1].attrib
-        assert '(Owner) added to ' in important_changes[1].text_content()
+        assert len(important_changes) == 5
+        assert 'class' not in important_changes[0].attrib
+        assert '(Owner) added to ' in important_changes[0].text_content()
+        assert 'class' not in important_changes[0].attrib
```

Comment:
This seems to be the exact same statement as on line 5340.

Reply:
Good catch, fixed in bcf464c3cb5c1667ca74a808b92099a52bfbc213